app/src/androidTest/java/ch/epfl/sdp/contamination/databaseIO/DataReceiverTest.java

```diff
@@ -39,38 +39,6 @@ public void getSickNeighborDoesGetIt() {
         sleep();
     }
 
-    class FakeGridInteractor extends GridFirestoreInteractor {
```

Comment:
where is this class now ?

Reply:
It was not used, but in case we need it later, I saved it to the wiki

Reply:
ok strange